packages/fields/src/fields/title/style.scss

```diff
@@ -0,0 +1,8 @@
+.fields-title-field span:first-child {
```

Comment:
Not a strong feeling, but CSS classes here are growing a bit disorganized.

We have:

```css
.fields-controls__featured_image {}
.fields-create-template-part-modal {}
.fields-title-field {}
```

We could try to consolidate our naming schema to `.package-function__name`:

```css
.fields-controls__featured-image {}
.fields-components__template-part-modal {}
.fields-field__title {}
.fields-field__pattern-title {}
```

Reply:
I updated the class names to be like `fields-field__${xxx}`.